oauth2_provider/models.py

```diff
@@ -216,6 +238,8 @@ def allow_scopes(self, scopes):
         """
         if not scopes:
             return True
+        if isinstance(scopes, six.string_types):
```

Comment:
When is `scopes` not a string? Are there steps that can be taken to ensure that it is always a string?